core-tests/src/test/java/org/mule/runtime/core/MuleEventTestCase.java

```diff
@@ -43,10 +38,7 @@
 
   @Test
   public void testEventSerialization() throws Exception {
-    FlowConstruct flowConstruct = MuleTestUtils.getTestFlow(muleContext);
-    Event event = Event.builder(DefaultEventContext.create(flowConstruct, TEST_CONNECTOR))
-        .message(InternalMessage.of("payload"))
-        .build();
+    Event event = eventBuilder().message(InternalMessage.of("payload")).build();
```

Comment:
Any reason this can't just be `Event event = event("payload");` or  `Event event = getTestEvent("payload")`?  I understand it makes use to use EventBuilder in a consistent way to how it's used in Mule for the non-trivial cases, but still don't see why we cant' simply the simplest cases further.  There is even a TEST_PAYLOAD or similar in AbstractMuleTestCase which can be used to create an event without even passing a payload.  Also ideally this event should be reused so that it can be asserted against and reused.